leetcodePython/SlidingWindow/slidingwindow_1234.py

```diff
@@ -0,0 +1,19 @@
+from collections import Counter
+
+
+class Solution:
+    def balancedString(self, s: str) -> int:
+        count = Counter(s)
+        left = 0
+        res = len(s)
+        require = len(s)//4
+        for right, ch in enumerate(s):
+            count[ch] -= 1
+            while left < len(s) and all(count[i] <= require for i in 'QWER'):
```

Comment:
Since we only have that `QWER` 4 kind of characters, for unbalanced string there are characters count higher than require. We use this to check balance